Mathlib/RingTheory/Localization/BaseChange.lean

```diff
@@ -190,3 +196,41 @@ lemma IsLocalizedModule.map_lTensor (g : M →ₗ[A] M') [h : IsLocalizedModule
   simp
 
 end
+
+namespace IsLocalization.Away
+
+variable {R S : Type*} [CommRing R] [CommRing S] [Algebra R S]
+    (r : R) (A : Type*) [CommRing A] [Algebra R A]
+
+instance tensor [IsLocalization.Away r A] :
+    IsLocalization.Away (algebraMap R S r) (S ⊗[R] A) := by
+  let _ : Algebra A (S ⊗[R] A) := Algebra.TensorProduct.rightAlgebra
+  simp only [IsLocalization.Away]
+  have : Submonoid.powers (algebraMap R S r) = Algebra.algebraMapSubmonoid S (.powers r) := by
+    simp [Algebra.algebraMapSubmonoid]
+  rw [this, Algebra.isLocalization_iff_isPushout _ A]
+  infer_instance
```

Comment:
This is more generally true for any submonoid (via `Algebra.algebraMapSubmonoid`) right? Can you separate that into a new lemma?

Reply:
Can you also provide `a ⊗t IsLocalization.mk x y = IsLocalization.mk (a ⊗t x) y` (or something like that?)